testkit/appkit/block_resp.go

```diff
@@ -0,0 +1,74 @@
+package appkit
+
+import "time"
+
+type BlockResp struct {
```

Comment:
As far as I understand, this code is autogenerated. I'd suggest to add a note above this struct: `this code is autogenerated. Please do not modify it manually ` or smth similar. You can also add a link to where it can be generated in case of any changes in json.

Reply:
Done